.github/workflows/testBuild.yml

```diff
@@ -6,22 +6,42 @@ on:
       PULL_REQUEST_NUMBER:
         description: Pull Request number for correct placement of apps
         required: true
-  pull_request_target:
-    types: [opened, synchronize, labeled]
-    branches: ['*ci-test/**']
+      WEB:
+        description: Should build web app?
+        type: boolean
+        default: true
+      DESKTOP:
+        description: Should build desktop app?
+        type: boolean
+        default: true
+      IOS:
+        description: Should build iOS app?
+        type: boolean
+        default: true
+      IOS_HYBRID:
+        description: Should build iOS hybrid app? 
+        type: boolean
+        default: true
+      ANDROID:
+        description: Should build android app? 
+        type: boolean
+        default: true
+      ANDROID_HYBRID:
+        description: Should build android hybrid app? 
+        type: boolean
+        default: true
```

Comment:
Those checks allows to select platforms for which app should be build. Not always all platforms are needed so this will help to reduce workflow execution time and cost